scraper/scraper/spiders/budget_exp_receipts.py

```diff
@@ -11,10 +11,10 @@ class BudgetExpendituresSpider(budget_base.ExpenditureBaseSpider):
     '''
```

Comment:
File name "budget_exp_receipts" can be renamed to "expenditure_receipts_budget_spider.py"